### PR TITLE
Embed the diesel database migrations in the `todo` example.

### PR DESCRIPTION
**Pull request by jebrosen**
With this change, `diesel_cli` will no longer be needed in order to prepare or run the example or tests, making the example easier to work with (and possibly reducing the time taken for CI builds?). It will still be necessary to install `diesel_cli` manually in order to add migrations or regenerate the schema.

### CODE REVIEW

File: examples/todo/src/main.rs [hunk 1]
```diff
@@ -19,6 +22,8 @@ use diesel::SqliteConnection;
 
 use task::{Task, Todo};
 
+embed_migrations!();
```
Comment (SergioBenitez) [inline examples/todo/src/main.rs]: What does this do, exactly? Perhaps a small comment above would help.

**Comment by jebrosen** [comment]
Another thought: it might even be possible to eliminate the bootstrap script entirely by running the tests against an in-memory database, by setting the url in the configuration to ":memory:".

**Comment by jebrosen** [comment]
@SergioBenitez With this set of changes, the bootstrap script now only deletes the database file. I planned to try making the bootstrap script entirely unnecessary -- maybe by using an in-memory database, database transactions, or a temporary file in test mode. Would you prefer to wait for those changes here, or for me to open a separate PR later?

**Comment by jebrosen** [comment]
Merged in 30cf753.

**Pull request closed without merging**